### PR TITLE
Optimal Changes

### PR DESCRIPTION
**Pull request by Mippy**
Spelling mistakes, unnecessary code, etc.

### CODE REVIEW

File: bible/bible.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def getBiblePassage(self, passage):
             em.add_field(name=i, value=o, inline=False)
 
         # Boop it to the user
-        await self.bot.say('', embed=em)
```
Comment (Mippy) [inline bible/bible.py]: Putting quotes here is unnecessary.

File: dpizza/dpizza.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from random import randint
 
 
-orderChannel = Object('266684197970640897')  # <---- place the chanenl ID there
```
Comment (Mippy) [inline dpizza/dpizza.py]: Simple misspelling

File: bible/bible.py (hunk 1 above)
Comment (atiwiex) [reply bible/bible.py]: Ok

File: dpizza/dpizza.py (hunk 2 above)
Comment (atiwiex) [reply dpizza/dpizza.py]: Pizza was a test cog but thanks

**Pull request merged**